tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from ansible_rulebook.condition_types import Condition as Conditions
+from ansible_rulebook.rule_types import (
+    Action,
+    Condition,
+    EventSource,
+    EventSourceFilter,
+    Rule,
+    RuleSet,
+)
+
+
+@pytest.fixture()
+def create_conditions(**kwargs):
+    def _conditions(**kwargs):
+        return Conditions(kwargs.pop("value", "2 > 1"))
+
+    return _conditions
+
+
+@pytest.fixture()
+def create_condition(create_conditions, **kwargs):
+    def _condition(**kwargs):
+        when = kwargs.pop("when", "all")
+        value = kwargs.pop("value", [create_conditions()])
+        return Condition(when, value)
+
+    return _condition
+
+
+@pytest.fixture
```

Comment:
@mkanoor What is the difference between `@pytest.fixture()` and `@pytest.fixture`?

Reply:
@bzwei They are inter changeable and uses the default scope

Reply:
@bzwei Nice catch, I have made them all consistent now.